### ISSUE TITLE
GLOB_ defines are missing from HHI

### ISSUE THREAD

**Issue by thewavelength** [facebook/hhvm#6677]
E. g. `GLOB_NOSORT` produces `Unbound name: GLOB_NOSORT (a global constant) (Naming[2049])`

```
$ hhvm --version
HipHop VM 3.11.0 (rel)
Compiler: tags/HHVM-3.11.0-0-g3dd564a8cde23e3205a29720d3435c771274085e
Repo schema: 52047bdda550f21c2ec2fcc295e0e6d02407be51
```

**Comment by Orvid** [comment]
A bit of clarification on this: The constants are defined in HHVM itself, but they aren't currently present in the HHI. (`builtins_file.hhi`)

**Comment by usox** [comment]
fixed within #7188
